3rdParty/velocypack/include/velocypack/Value.h

```diff
@@ -51,21 +51,23 @@ class Value {
     UInt64 = 4,
     String = 5,
     CharPtr = 6,
-    VoidPtr = 7
+    VoidPtr = 7,
+    StringView = 8
   };
 
  private:
   ValueType _valueType;
   CType _cType;  // denotes variant used, 0: none
 
   union {
-    bool b;                // 1: bool
-    double d;              // 2: double
-    int64_t i;             // 3: int64_t
-    uint64_t u;            // 4: uint64_t
-    std::string const* s;  // 5: std::string
-    char const* c;         // 6: char const*
-    void const* e;         // 7: external
+    bool b;                      // 1: bool
+    double d;                    // 2: double
+    int64_t i;                   // 3: int64_t
+    uint64_t u;                  // 4: uint64_t
+    std::string const* s;        // 5: std::string
+    char const* c;               // 6: char const*
+    void const* e;               // 7: external
+    std::string_view const* sv;  // 8: std::string_view
```

Comment:
using std::string_view unconditionally will break VelocyPack c++11 and c++14 builds.

Reply:
Right, thanks. Will fix it.